doc/en/source/scorpioIntegration.rst

```diff
@@ -14,11 +14,19 @@ Integration steps
 
 **Pre-Requisites:**
 
```

Comment:
[Major] Improve the diagram and write more information except only NGSI-LD data write which request like NOTIFY, PATCH, UPDATE request to make it more meaningful.